pykickstart/commands/interactive.py

```diff
@@ -56,3 +56,9 @@ def _getParser(self):
         op = FC3_Interactive._getParser(self)
         op.description += "\n\n.. deprecated:: %s" % versionToLongString(F14)
         return op
+
+class F15_Interactive(RemovedCommand):
+    def _getParser(self):
+        op = F14_Interactive._getParser(self)
+        op.description += "\n\n.. versionremoved:: %s" % versionToLongString(F15)
```

Comment:
I wonder why this have to be specified manually and can't be implemented by the ``RemovedCommand``.

Reply:
Seems like a job for a macro :P (joking)

Reply:
The class doesn't know how or where to get the previous _getParser from. I tried playing with some mixins a bit and it just ended up being a mess and not working  completely.

Doing it this way also matches how deprecation is handled so it's easier for infrequent contributors to understand.